code/defines/procs/dbcore.dm

```diff
@@ -72,12 +72,21 @@ DBConnection/proc/NewQuery(sql_query,cursor_handler=src.default_cursor) return n
 
 
 DBQuery/New(sql_query,DBConnection/connection_handler,cursor_handler)
+	if(IsAdminAdvancedProcCall())
+		to_chat(usr, "<span class='boldannounce'>DB query blocked: Advanced ProcCall detected.</span>")
+		message_admins("[key_name(usr)] attempted to create a DB query via advanced proc-call")
+		log_admin("[key_name(usr)] attempted to create a DB query via advanced proc-call")
+		return
```

Comment:
Shouldn't it qdel itself right after this? Same goes for the other News

Reply:
Not sure if its needed, will check with maints

Reply:
Spoke with the maints, these things will sort themselves out in the GC